components/brave_shields/core/browser/ad_block_default_resource_provider.cc

```diff
@@ -77,8 +95,17 @@ void AdBlockDefaultResourceProvider::LoadResources(
   base::ThreadPool::PostTaskAndReplyWithResult(
       FROM_HERE, {base::MayBlock()},
       base::BindOnce(&brave_component_updater::GetDATFileAsString,
-                     component_path_.AppendASCII(kAdBlockResourcesFilename)),
+                     resources_path),
       std::move(cb));
 }
 
+void CheckAdBlockExceptionComponentsUpdate() {
+  base::SequencedTaskRunner::GetCurrentDefault()->PostDelayedTask(
+      FROM_HERE, base::BindOnce([]() {
```

Comment:
`base::BindOnce(brave_component_updater::BraveOnDemandUpdater::GetInstance(), &brave_component_updater::BraveOnDemandUpdater::OnDemandUpdate, kAdblockExceptionComponentId);`

Instead of putting it in a lambda

Reply:
I can't seem to get this exactly as you put it. I believe the order should be function, instance and variable:
```
base::BindOnce(&brave_component_updater::BraveOnDemandUpdater::OnDemandUpdate,
                     base::Unretained(brave_component_updater::BraveOnDemandUpdater::GetInstance()),
                     kAdBlockExceptionComponentId),
```

Also not sure about that base::Unretained. I think its fine because its a shared instance. But im forced to do something there.

Reply:
That looks correct.